src/components/Dropdown/Dropdown.spec.js

```diff
@@ -18,32 +19,73 @@ const data = [
     }
 ];
 
+const dataGrouped = [
```

Comment:
What do you think to put all data in `src/data` to avoid repeating the same fake data object for storybook and tests (like we did for the TreeView)?

Reply:
I think we need some of the test data in the story itself, otherwise, it will be difficult for developers implementing the component to see what the data format should be - they would have to leave storybook and go to github to check the source code.